deis-dev/tpl/deis-objectstorage-secret.yaml

```diff
@@ -8,7 +8,9 @@ metadata:
   annotations:
     deis.io/objectstorage: "{{.storage}}"
 type: Opaque
-data: {{ if eq .storage "gcs" }}
+data: {{ if eq .storage "minio" }}
+  access-key-id: OFRaUlkySlJXTVBUNlVNWFI2STUK
```

Comment:
No.This was on purpose because i think we need to use one secret for object store instead of two.Slowly we can remove the usage of minio-user secret from everywhere and remove the minio-user secret from the charts.